specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-01-01/machineLearningServices.json

```diff
@@ -0,0 +1,6522 @@
+          "type": "integer",
+          "format": "int64",
+          "description": "The current usage of the resource."
+        },
+        "limit": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int64",
+          "description": "The maximum permitted usage of the resource."
+        },
+        "name": {
+          "readOnly": true,
+          "$ref": "#/definitions/UsageName",
+          "description": "The name of the type of usage."
+        }
+      },
+      "description": "Describes AML Resource Usage."
+    },
+    "ListUsagesResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Usage"
+          },
+          "description": "The list of AML resource usages."
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page of AML resource usage information. Call ListNext() with this to fetch the next page of AML resource usage information."
+        }
+      },
+      "description": "The List Usages operation response."
+    },
+    "VirtualMachineSize": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "title": "Virtual Machine size name",
+          "description": "The name of the virtual machine size.",
+          "readOnly": true
+        },
+        "family": {
+          "type": "string",
+          "title": "Virtual Machine family name",
+          "description": "The family name of the virtual machine size.",
+          "readOnly": true
+        },
+        "vCPUs": {
+          "type": "integer",
+          "format": "int32",
+          "title": "Number of vPUs",
+          "description": "The number of vCPUs supported by the virtual machine size.",
+          "readOnly": true
+        },
+        "gpus": {
+          "type": "integer",
+          "format": "int32",
+          "title": "Number of gPUs",
+          "description": "The number of gPUs supported by the virtual machine size.",
+          "readOnly": true
+        },
+        "osVhdSizeMB": {
+          "type": "integer",
+          "format": "int32",
+          "title": "OS VHD Disk size",
+          "description": "The OS VHD disk size, in MB, allowed by the virtual machine size.",
+          "readOnly": true
+        },
+        "maxResourceVolumeMB": {
+          "type": "integer",
+          "format": "int32",
+          "title": "Resource volume size",
+          "description": "The resource volume size, in MB, allowed by the virtual machine size.",
+          "readOnly": true
+        },
+        "memoryGB": {
+          "type": "number",
+          "format": "double",
+          "title": "Memory size",
+          "description": "The amount of memory, in GB, supported by the virtual machine size.",
+          "readOnly": true
+        },
+        "lowPriorityCapable": {
```

Comment:
Consider changing this and premiumIO to enum { True, False } for future-proofing.

Reply:
May I ask why enum { True, False } is better than boolean type? Also, since this definition is carried over from previous versions, is it okay to keep it this way?